runtime/tests/port/j9strTest.c

```diff
@@ -1604,35 +1614,71 @@ j9str_test_atoe_vsnprintf(struct J9PortLibrary *portLibrary)
 
 	/* Neither min_width nor precision is specified for a null terminated input string*/
 	j9tty_printf(PORTLIB, "\n\tTesting case 1\n");
-	rc |= printToBuffer(portLibrary, TRUE, buffer0, bufferLength0, expectedOutput1, "%s", nullTerminatedString);
+	rc |= printToBuffer(portLibrary, TRUE, buffer0, bufferLength0, expectedOutput1, strlen(expectedOutput1), "%s", nullTerminatedString);
 
 	/* min_width is less than the length of inputString */
 	j9tty_printf(PORTLIB, "\n\tTesting case 2\n");
-	rc |= printToBuffer(portLibrary, FALSE, buffer2, bufferLength2, expectedOutput2, "%*s", 2, nonNullTerminatedString);
+	rc |= printToBuffer(portLibrary, FALSE, buffer2, bufferLength2, expectedOutput2, strlen(nullTerminatedString), "%*s", 2, nullTerminatedString);
```

Comment:
I think `expectedOutput1` should be expected here.

Reply:
It stops writing to the buffer when the `count` has reached therefore the buffer would be "TES". It will continue the estimation without writing to the buffer.